### DISCUSSION TITLE
执行数据集划分无反应

### DISCUSSION THREAD

**Discussion by caishenzhishou** [PaddlePaddle/PaddleOCR#14080] [Q&A]
执行如下命令python gen_ocr_train_val_test.py --trainValTestRatio 6:2:2 --datasetRootPath ../train_data
![image](https://github.com/user-attachments/assets/b0de2f9b-69dc-4d49-8b95-1ad68d58abc8)
执行后，未生成rec和det文件夹。
哪位大佬给个建议，哪里出问题了？

**Comment by GreatV** [comment]
根据你提供的信息，执行命令后没有生成`rec`和`det`文件夹，可能的问题有几个：

1. 检查`../train_data`路径是否正确，确保该路径存在并且里面有数据。
2. 确保`gen_ocr_train_val_test.py`脚本没有错误，检查脚本内是否有任何提示信息或错误日志。
3. 确认你是否有权执行该脚本和写入文件夹的权限。
4. 检查Python环境是否配置正确，相关的依赖库是否已安装。
5. 尝试使用不同的命令参数看是否有不同的输出。

**Reply by kongxa** [reply]
检查是否缺少依赖库，缺少则安装。gen_ocr_train_val_test.py包含4个组件os、shutil、random、argparse。如果缺少os则需要重新安装python。

        pip install shutilwhich
        pip install random2
        pip install argparse
        安装最新版本的python：
        https://www.python.org/ftp/python/3.13.1/python-3.13.1-amd64.exe
